tests/network/functional/bridge_test.py

```diff
@@ -71,7 +71,7 @@ def test_add_bridge_with_custom_opts(self, adapter, switch, nic0):
             'nic': nic0,
             'switch': switch,
             'custom': {
-                'bridge_opts': 'multicast_snooping=0 multicast_router=0'
+                'bridge_opts': 'multicast_snooping=False'
```

Comment:
This must be compatible with the old way, that's also the reason why it fails the validation step in tests.

Reply:
you mean we should pass `0` or `1` and convert it to True or False if needed?

Reply:
Yes